sippy-ng/src/prow_job_runs/ProwJobRun.js

```diff
@@ -0,0 +1,926 @@
+import * as lodash from 'lodash'
+import { Button, ButtonGroup, TextField } from '@material-ui/core'
+import { Error } from '@material-ui/icons'
+import { useHistory } from 'react-router-dom'
+import Alert from '@material-ui/lab/Alert'
+import PropTypes from 'prop-types'
+import React, { Fragment, useEffect, useState } from 'react'
+import TimelineChart from '../components/TimelineChart'
+
+export default function ProwJobRun(props) {
+  const history = useHistory()
+
+  const [fetchError, setFetchError] = React.useState('')
+  const [isLoaded, setLoaded] = React.useState(false)
+  const [eventIntervals, setEventIntervals] = React.useState([])
+  const [filteredIntervals, setFilteredIntervals] = React.useState([])
+
+  // TODO: Not sure this needs to be a useState param, really just need a constant map but
+  // wasn't working.
+  const [allCategories, setAllCategories] = useState(
+    new Map([
+      ['operator_unavailable', 'Operator Unavailable'],
+      ['operator_progressing', 'Operator Progressing'],
+      ['operator_degraded', 'Operator Degraded'],
+      ['pods', 'Pods (careful)'],
+      ['pod_logs', 'Pod Logs'],
+      ['interesting_events', 'Interesting Events'],
+      ['alerts', 'Alerts'],
+      ['node_state', 'Node State'],
+      ['e2e_test_failed', 'E2E Failed'],
+      ['e2e_test_flaked', 'E2E Flaked'],
+      ['e2e_test_passed', 'E2E Passed'],
+      ['disruption', 'Disruption'],
+    ])
+  )
```

Comment:
```js
const allCategories = {
  operator_unavailable: 'Operator Unavailable',
  operator_progressing: 'Operator Progressing',
  operator_degraded: 'Operator Degraded',
  pods: 'Pods (careful)',
  pod_logs: 'Pod Logs',
  interesting_events: 'Interesting Events',
  alerts: 'Alerts',
  node_state: 'Node State',
  e2e_test_failed: 'E2E Failed',
  e2e_test_flaked: 'E2E Flaked',
  e2e_test_passed: 'E2E Passed',
  disruption: 'Disruption'
};
```

didn't work?

Reply:
I don't know what I did, but this works, thanks!